app/javascript/packages/document-capture/components/mobile-intro-step.jsx

```diff
@@ -1,16 +1,33 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import ServiceProviderContext from '../context/service-provider';
 import useI18n from '../hooks/use-i18n';
 
 function MobileIntroStep() {
-  const { t } = useI18n();
+  const { t, formatHTML } = useI18n();
+  const serviceProvider = useContext(ServiceProviderContext);
 
   return (
     <>
       <p>{t('doc_auth.info.document_capture_intro_acknowledgment')}</p>
       <p>
-        <a href="/verify/jurisdiction/errors/no_id">{t('idv.messages.jurisdiction.no_id')}</a>
+        {formatHTML(t('doc_auth.info.id_worn_html'), {
+          strong: 'strong',
+        })}{' '}
+        <a href="/">{t('doc_auth.info.accepted_ids')}</a>
```

Comment:
just wanted to highlight this as another pending URL we'll need to change

Reply:
@aduth following up on the "Full list of state-issued IDs" link - since this URL/content doesn't currently exist, we can omit it for now, and add it to this screen as an improvement when it's ready.  Thanks!